projects/addon-mobile/internal/primitive-calendar-mobile/primitive-calendar-mobile.template.html

```diff
@@ -11,8 +11,9 @@
             class="t-cell"
             [class.t-cell_today]="itemIsToday(sheet[rowIndex][colIndex])"
             [class.t-cell_interval]="itemIsInterval(sheet[rowIndex][colIndex])"
-            [attr.data-tui-element-state]="getItemState(sheet[rowIndex][colIndex])"
-            [attr.data-tui-element-range]="getItemRange(sheet[rowIndex][colIndex])"
+            [class.t-cell_disabled]="itemIsDisabled(sheet[rowIndex][colIndex])"
```

Comment:
I think we don't need `t-cell_disabled` if we can already tell that the cell is disabled by `[data-state="disabled"]`. Please see if we can refactor this a bit because that is one less binding/getter per each day.

Reply:
fixed